Gemfile

```diff
@@ -4,8 +4,10 @@ gem 'spree', github: 'godaddy/spree', branch: '2-2-nemo-stable'
 gem 'spree_auth_devise', github: 'spree/spree_auth_devise', :branch => '2-2-stable'
 
 group :test do
-  gem 'rails', '4.0.9' #Temporary version lock to 4.0.9 to avoid bad rails version.
+  gem 'rails'
+  gem 'rspec'
   gem 'rspec-rails', '~> 2.0'
+  gem 'sass-rails'
```

Comment:
I was about to comment on how sass-rails is already in the gemspec as a dev dependency and we don't need it here, but then I removed it and everything broke even more than it was already broken.  Welp.